lib/prefill.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    class MonthlyPrefillService < ApplicationService
+      SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+      Error = Exceptions::AppInternalError.with_scope('declarations.prefills.prefill_service')
+
+      class UnsupportedDeclaration < Error; end
+
+      private
+
+      def initialize(declaration:)
```

Comment:
The `initialize` method should take the company as the first argument, and the main service resource as the second. Please refactor the method signature to follow the convention.
```suggestion
 def initialize(company, declaration:)
   @company = company
   @declaration = declaration
 end
```